recipes/screenFetch/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "screenFetch" %}
+{% set version = "3.9.1" %}
+{% set author = "KittyKatt" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ author }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: aa97dcd2a8576ae18de6c16c19744aae1573a3da7541af4b98a91930a30a3178
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - coreutils
+  run:
+    - bash
+
+test:
+  commands:
+    - screenfetch -V
+
+about:
+  home: https://github.com/{{ author }}/{{ name }}
```

Comment:
Please don't use jinja templates here. It obscure the links, make them unclickable and doesn't really help the recipe.

Reply:
Thank you for reviewing.

I'll change it.